tests/test_errors.py

```diff
@@ -122,16 +122,16 @@ def no_bar(foo=42):
     # all attempts to pass `bar` should fail
     est = wrapper(model=no_bar, model__bar=42)
     with pytest.raises(TypeError, match="got an unexpected keyword argument"):
-        est.fit([[1]], [1])
+        est.fit(np.array([[0], [1]]), np.array([0, 1]))
```

Comment:
Doesn't passing lists of numbers work? Scikit-learn supports that interface.

Reply:
The change from list -> np.array is not strictly necessary. What is necessary is `[1]` -> `[0, 1]` because `type_of_target([1])=="unknown"`. I'll _try_ to revert all of the unnecessary changes.

Reply:
I reverted all of this to lists, hopefully I didn't miss any.